src/actions/types.ts

```diff
@@ -1,81 +1,82 @@
-}
+  chainId: number;
+  contractAddress?: Address;
+  tokenIn?: Address;
+  tokenOut?: Address;
+  amountIn?: bigint | FilterOperator;
+  amountOut?: bigint | FilterOperator;
+  recipient?: Address;
+  deadline?: bigint | FilterOperator;
+};
 
 export type OptionsActionParams = {
-  chainId: number
-  contractAddress?: Address
-  token?: Address
-  amount?: bigint | FilterOperator
-  recipient?: Address
-  orderType?: OrderType
-}
+  chainId: number;
+  contractAddress?: Address;
+  token?: Address;
+  amount?: bigint | FilterOperator;
+  recipient?: Address;
+  orderType?: OrderType;
+};
 
 export type StakeActionParams = {
-  chainId: number
-  contractAddress?: Address
-  tokenOne?: Address
-  amountOne?: bigint | FilterOperator
-  tokenTwo?: Address
-  amountTwo?: bigint | FilterOperator
-  duration?: bigint | FilterOperator
-}
+  chainId: number;
+  contractAddress?: Address;
+  tokenOne?: Address;
+  amountOne?: bigint | FilterOperator;
+  tokenTwo?: Address;
+  amountTwo?: bigint | FilterOperator;
+  duration?: bigint | FilterOperator;
+};
 
 export type BridgeActionParams = {
-  sourceChainId: number
-  destinationChainId: number
-  contractAddress?: Address
-  tokenAddress?: Address
-  amount?: bigint | FilterOperator
-  recipient?: Address
-}
+  sourceChainId: number;
+  destinationChainId: number;
+  contractAddress?: Address;
+  tokenAddress?: Address;
+  amount?: bigint | FilterOperator;
+  recipient?: Address;
+};
 
 export type MintActionParams = {
-  chainId: number
-  contractAddress: Address
-  tokenId?: number
-  amount?: number | FilterOperator
-  recipient?: Address
-}
+  chainId: number;
+  contractAddress: Address;
+  tokenId?: number;
+  amount?: number | FilterOperator;
+  recipient?: Address;
+};
 
-export type BurnActionParams = MintActionParams
+export type BurnActionParams = MintActionParams;
 
 export type QuestActionParams = {
-  chainId: number
-  rewardToken?: Address
-  rewardAmount?: bigint | FilterOperator
-  startTime?: bigint | FilterOperator
-  endTime?: bigint | FilterOperator
-  totalParticipants?: bigint | FilterOperator
-  actionSpec?: string
-}
+  chainId: number;
+  rewardToken?: Address;
+  rewardAmount?: bigint | FilterOperator;
+  startTime?: bigint | FilterOperator;
+  endTime?: bigint | FilterOperator;
+  totalParticipants?: bigint | FilterOperator;
+  actionSpec?: string;
+};
 
 export type DelegateActionParams = {
-  chainId: number
-  delegate?: Address
-  project: Address | string
-  contractAddress?: Address
-  amount?: bigint | FilterOperator
-  delegator?: Address
-}
+  chainId: number;
+  delegate?: Address;
+  project: Address | string;
+  contractAddress?: Address;
+  amount?: bigint | FilterOperator;
+  delegator?: Address;
+};
 
 export type VoteActionParams = {
-  chainId: number
-  governorAddress?: Address
-  proposalId?: bigint | FilterOperator
-  support?: bigint | FilterOperator
-}
+  chainId: number;
+  project: Address | string;
+  contractAddress: Address;
```

Comment:
Looks good to me.

I think `contractAddress` might work better as an optional parameter.